src/renderer/components/recording-page/recording-page.test.tsx

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
Please rebase after #85 is merged and also modify the style of this test following that too.

Reply:
Should I hold off on pushing any commits to this branch until 85 is merged?

Reply:
Yes that's what I meant. And it's merged already, so you are free to rebase now.